frontend/src/console/actions.ts

```diff
@@ -164,8 +165,14 @@ export interface CancelDeploymentFailure extends Action {
   readonly errors: any;
 }
 
+export interface UserRefreshData extends Action {
+  readonly type: Types.CLEAR_CONSOLE_DATA;
+  readonly data: Array<InstanceSettingsModel>;
+}
+
 export type ActionTypes =
   | ClearFeedbackMessage
+  | UserRefreshData
```

Comment:
Here this is still `UserRefreshData` instead of something related to `ClearConsoleData`. 
`UserRefreshData` isn't representative of what this function does.

Make sure this doesn't conflict with the `clearConsoleData` function below.

Reply:
Refactored the code and `UserRefreshData` changed to `ClearConsoleData` for better understanding(missed in the last commit, Thanks for this)